src/diffusers/pipelines/stable_diffusion/pipeline_onnx_stable_diffusion.py

```diff
@@ -198,3 +198,27 @@ def __call__(
             return (image, has_nsfw_concept)
 
         return StableDiffusionPipelineOutput(images=image, nsfw_content_detected=has_nsfw_concept)
+
+
+class StableDiffusionOnnxPipeline(OnnxStableDiffusionPipeline):
+    def __init__(
+        self,
+        vae_decoder: OnnxRuntimeModel,
+        text_encoder: OnnxRuntimeModel,
```

Comment:
Wanted to do just `*args, **kwargs` but that broke `register_modules()`, so we're repeating the names here.